packages/visx-demo/next.config.js

```diff
@@ -1,22 +1,9 @@
 /* eslint-disable no-undef */
-const transpileModules = require('next-transpile-modules');
 const path = require('path');
 
 const isProd = process.env.NODE_ENV === 'production';
 
-// these deps are ESM-only so we must compile them even
-// though we also use experimental.esmExternals
-const withTM = transpileModules([
```

Comment:
this is testing the core fix for https://github.com/airbnb/visx/issues/1637, removing ESM issues in next apps